man/userdbctl.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="userdbctl" conditional='ENABLE_USERDB'
+    xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>userdbctl</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>userdbctl</refentrytitle>
+    <manvolnum>1</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>userdbctl</refname>
+    <refpurpose>Inspect users, groups and group memberships</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <cmdsynopsis>
+      <command>userdbctl</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="req">COMMAND</arg>
+      <arg choice="opt" rep="repeat">NAME</arg>
+    </cmdsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><command>userdbctl</command> may be used to inspect user and groups (as well as group memberships)
+    of the system. This client utility inquires user/group information provided by various system services,
+    both operating on advanced JSON user/group records (as defined by the <ulink
+    url="https://systemd.io/USER_RECORD">JSON User Record</ulink> and <ulink
+    url="https://systemd.io/GROUP_RECORD">JSON Group Record</ulink> definitions), and classic UNIX NSS/glibc
+    user and group records. This tool is primarily a client to the <ulink
+    url="https://systemd.io/USER_GROUP_API">User/Group Record Lookup API via Varlink</ulink>.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>Options</title>
+
+    <para>The following options are understood:</para>
+
+    <variablelist>
+
+      <varlistentry>
+        <term><option>--display=</option><replaceable>MODE</replaceable></term>
```

Comment:
This is normally called --output in other tools, e.g. journalctl.

Reply:
I thought about this, but I wasn't so sure. Those tools that support --output= do so because they do log output, i.e. systemctl, journalctl, machinectl and loginctl. The --output= directly translates to the log output fashioning. Now here this doesn't really have something to do with log output but something inherently different: display of user info.

We have another tool where the display mode is switched with a diferent switch that --output=: busctl, it uses --json=, i.e. very different again. I was thinking about going for that, but felt that it was too restricting in what we can do with this.

Anyway, this is definitely something to discuss and figure out before the release. But I am tempted to just get this quickly in, we can still rename this.

The question to ask is I guess is what is a higher good: that --output= takes the same arguments on all our tools (i.e. "short", "short-precise", "short-iso", …) and primarily applies to logs, or that selecting the display mode is always done with the same switch, regardless what the set of modes actually entails.

Reply:
I'd vote for the same switch, because tab-completion can be used to fill in the output mode, once the name of the option itself is entered.

Reply:
ok, will rework